rero_mef/places/mef/mappings/v7/places_mef/mef-place-v0.0.1.json

```diff
@@ -119,6 +147,9 @@
                   },
                   "value": {
                     "type": "keyword"
+                  },
+                  "identifier": {
```

Comment:
How about calling this field `_text`? This way we know that it is constructed and specific to MEF and not to be confused with `identifiedBy.value`.